_posts/en/newsletters/2018-10-09-newsletter.md

```diff
@@ -0,0 +1,283 @@
+    meaning their proof of work during that brief period of time is
+    wasted (it neither increases network security nor provides the
+    miners with financial compensation).  Miners with more hash rate
+    produce more blocks on average, so they get the head start more
+    often and less of their proof of work is wasted.
+
+    For two perfectly fair miners half a world apart, the network delay
+    can be about one second, meaning a small miner far away from most
+    other miners is likely to only be productive for 599 of the average
+    of 600 seconds (ten minutes) between blocks.  A 1/600 loss of
+    efficiency (0.16%) is not too bad, but if the block frequency were
+    increased, the loss of efficiency would increase also: at one block
+    per minute, the loss of efficiency would be 1.66%; at one block per
+    six seconds, 16.66%.
+
+    The small miner could increase his efficiency by moving closer to
+    other miners or even completely eliminate the efficiency loss by
+    merging with them, but this is the mining centralization that it's
+    essential to avoid in Bitcoin if we want to prevent miners from
+    being able to effectively restrict (censor) which transactions are
+    included in blocks.
+
+{% include references.md %}
+
+[videos]: https://tokyo2018.scalingbitcoin.org/#remote-participation
+[transcripts]: http://diyhpl.us/wiki/transcripts/scalingbitcoin/tokyo-2018/
+[vid subsidy]: https://youtu.be/y8hJ0VTPE34?t=39
+[tx subsidy]: http://diyhpl.us/wiki/transcripts/scalingbitcoin/tokyo-2018/playing-with-fire-adjusting-bitcoin-block-subsidy/
+[vid forward blocks]: https://youtu.be/y8hJ0VTPE34?t=3744
+[tx forward blocks]: http://diyhpl.us/wiki/transcripts/scalingbitcoin/tokyo-2018/forward-blocks/
+[paper forward blocks]: http://freico.in/forward-blocks-scalingbitcoin-paper.pdf
+[vid bls msig]: https://youtu.be/IMzLa9B1_3E?t=29
+[tx bls msig]: http://diyhpl.us/wiki/transcripts/scalingbitcoin/tokyo-2018/compact-multi-signatures-for-smaller-blockchains/
+[paper bls msig]: https://eprint.iacr.org/2018/483.pdf
+[vid accumulators]: https://youtu.be/IMzLa9B1_3E?t=3522
+[tx accumulators]: http://diyhpl.us/wiki/transcripts/scalingbitcoin/tokyo-2018/accumulators/
+[vid scriptless ecdsa]: https://youtu.be/3mJURLD2XS8?t=3624
+[tx scriptless ecdsa]: http://diyhpl.us/wiki/transcripts/scalingbitcoin/tokyo-2018/scriptless-ecdsa/
+[tx script]: http://diyhpl.us/wiki/transcripts/scalingbitcoin/tokyo-2018/bitcoin-script/
+[musig paper]: https://eprint.iacr.org/2018/068
+[schnorr pre-bip]: https://github.com/sipa/bips/blob/bip-schnorr/bip-schnorr.mediawiki
+[pairing-based cryptography]: https://en.wikipedia.org/wiki/Pairing-based_cryptography 
+[bls sigs]: https://en.wikipedia.org/wiki/Boneh%E2%80%93Lynn%E2%80%93Shacham
+[forced fork]: https://petertodd.org/2016/forced-soft-forks
+[scriptless scripts transcript]: https://scalingbitcoin.org/transcript/stanford2017/using-the-chain-for-what-chains-are-good-for
```

Comment:
That's not the best link for a transcript about scriptless scripts; I like these:

* https://diyhpl.us/wiki/transcripts/layer2-summit/2018/scriptless-scripts/
* https://diyhpl.us/wiki/transcripts/realworldcrypto/2018/mimblewimble-and-scriptless-scripts/

Reply:
Actually, nevermind. It's not so bad. It just has no headings.